CakeMap.Rmd

```diff
@@ -16,7 +16,7 @@ different parts of Leeds, UK. The example is deliberately rather absurd to make
 it more memorable. The steps are presented in a generalisable way, to be
 applicable to a wide range of datasets.
 
-The input microdataset is a randomised ('jumbled') subset of the
+The input microdataset is a randomized ('jumbled') subset of the
```

Comment:
My version was British English, yours is American. I don't mind which we use; we should be consistent.